images/resource-consumer/consume-cpu/consume_cpu_windows.go

```diff
@@ -72,15 +79,15 @@ func main() {
 	handle := syscall.Handle(phandle)
 	flag.Parse()
 	// convert millicores to percentage
-	millicoresPct := float64(*millicores) / float64(10)
+	targetPct := float64(*millicores) / float64(10)
 	duration := time.Duration(*durationSec) * time.Second
 	start := time.Now()
 	first := statsNow(handle)
 
 	for time.Now().Sub(start) < duration {
 		cpu := usageNow(first, statsNow(handle))
 		fmt.Println("cpu: ", cpu)
```

Comment:
This is actually a time consuming action. Because time is spent executing this, the CPU utilization does not reach the desired values:

    PS C:\workspace\k8s_images\images\resource-consumer\consume-cpu> .\consume_cpu_windows.exe -duration-sec 20 -millicores 1000
    cpu:  NaN
    cpu:  0
    cpu:  17.362230010378447
    cpu:  17.759334590410074
    cpu:  19.17306518719456
    cpu:  17.530168719158922
    cpu:  17.937045275972206
    cpu:  19.612749144674936
    cpu:  20.253901989447044
    cpu:  19.53195966120102
    cpu:  19.054819954817212
    cpu:  20.09014672057374
    cpu:  19.792669495254426
    cpu:  20.508622590592328
    cpu:  20.316400379483763
    cpu:  21.211256649870368
    cpu:  21.159277352213714
    cpu:  21.23777335320121
    cpu:  21.294004397855133
    cpu:  21.2304964770284
    cpu:  21.14616192195179

NOTE: my env has 4 cores, so the target value should have been 25. Removing this print and watching the CPU utilization in task manager, I can see that it peaks at 25%.

We should probably remove this print.